gribscan/tools.py

```diff
@@ -10,9 +11,18 @@
 from .magician import MAGICIANS
 
 
+logging.basicConfig()
+
+
```

Comment:
This shouldn't be on module level, but rather called within the functions. We likely only want to default-configure the logger, when we are actually running as a standalone tool, but not when we are merely importing any of the functionality.

Reply:
My reasoning was that we never import any of the `tools` if we don't want to run it as a standalone tool. But I am also fine with configuring the logger in both functions ☺️